lapis/src/test/kotlin/org/genspectrum/lapis/controller/MultiSegmentedSequenceControllerTest.kt

```diff
@@ -215,14 +216,6 @@ class MultiSegmentedSequenceControllerTest(
         description: String,
         request: (String) -> MockHttpServletRequestBuilder,
     ) {
-        every {
```

Comment:
Sorry if Im missing it - why is this mocking nolonger required? In my understanding this mocks the `getGenomicSequence` function call and returns mocked data using `MockDataForEndpoints` - why can we remove this?

Reply:
Ah! Good catch thanks for explaining!